plugin.video.mazacka/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.mazacka"
+version="1.0"
```

Comment:
it is a good practice to have the version in the format: major.minor.revison, i.e., 1.0.0